test/jdk/javax/net/ssl/templates/TLSBase.java

```diff
@@ -168,43 +163,45 @@ static class Server extends TLSBase {
                 ssock.setReuseAddress(true);
                 ssock.setNeedClientAuth(builder.clientauth);
                 serverPort = ssock.getLocalPort();
-                System.out.println("Server Port: " + serverPort);
+                System.err.println("Server Port: " + serverPort);
             } catch (Exception e) {
                 System.err.println("Failure during server initialization");
                 e.printStackTrace();
             }
 
             // Thread to allow multiple clients to connect
-            t = new Thread(() -> {
+            new Thread(() -> {
                 try {
-                    while (true) {
+                    System.err.println("Server starting to accept");
+                    serverLatch.countDown();
+                    do {
                         SSLSocket sock = (SSLSocket)ssock.accept();
                         threadPool.submit(new ServerThread(sock));
-                    }
+                    } while (true);
                 } catch (Exception ex) {
                     System.err.println("Server Down");
                     ex.printStackTrace();
                 } finally {
                     threadPool.close();
                 }
-            });
-            t.start();
+            }).start();
         }
 
         class ServerThread extends Thread {
             SSLSocket sock;
 
             ServerThread(SSLSocket s) {
                 this.sock = s;
-                System.err.println("ServerThread("+sock.getPort()+")");
+                System.err.println("(Server) client connection on port " +
+                    sock.getPort());
                 clientMap.put(sock.getPort(), sock);
             }
 
             public void run() {
                 try {
                     write(sock, read(sock));
                 } catch (Exception e) {
-                    System.out.println("Caught " + e.getMessage());
+                    System.err.println("Caught " + e.getMessage());
```

Comment:
`e.printStackTrace()` will include the exception message. Do we need this line at all?

Reply:
true

Reply:
Actually the line change may not be necessary.  It would be good for stdout to get the message, but the stacktrace to be left for err.